pipelines/pipeline.yml

```diff
@@ -44,9 +44,15 @@ stages:
             displayName: "Set up Golang CI Tools"
 
           # Unit Tests
-          - script: |
-              make test
+          - task: AWSShellScript@1
             displayName: "Unit Tests"
+            inputs:
+              awsCredentials: "AWS_Login"
```

Comment:
@kddejong what's the story here?
I think we actually need to revert this -- I'll message you in Flowdock with details

Reply:
band-aid fix put it. We'll have to revisit this, if we start getting some community PRs